### DISCUSSION TITLE
What is the ETA of the v10 release?

### DISCUSSION THREAD

**Discussion by OrhanTozan** [xyflow/xyflow#1627] [Q&A] [answered]
What is the ETA of the v10 release?

**Comment by moklick** [comment, marked as answer]
We will release a documented beta v10 in November.

**Reply by OrhanTozan** [reply]
It's now almost January. Any updates on that documented beta?

**Reply by moklick** [reply]
"It's now almost January" ? What's the bossy tone here? You can find the installation instructions and changes in this PR: #1555 as the readme points out.
